assets/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+image=${1:-pulp/pulp:latest}
+
+mkdir settings
+echo "CONTENT_ORIGIN='http://localhost:8080'" >> settings/settings.py
+docker run --detach \
+           --name pulp \
+           --publish 8080:80 \
+           --volume "/$(pwd)/settings:/etc/pulp:Z" \
+           --device /dev/fuse \
+           "$image"
+sleep 10
+for _ in $(seq 10)
+do
+  sleep 3
+  if curl --fail http://localhost:8080/pulp/api/v3/status/ > /dev/null 2>&1
+  then
+    break
+  fi
+done
+curl --fail http://localhost:8080/pulp/api/v3/status/
+echo "SUCCESS: $image passed."
```

Comment:
`docker rm pulp`
`stop` or `halt` may be needed first. But should we not free the name pulp for the next container?

Reply:
I've tried to keep it simple by only running one container.

Reply:
Added stop and rm in a cleanup trap.